code/src/index.css

```diff
@@ -1,13 +1,114 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  margin: 20px;
+  font-family: "Roboto Mono", monospace;
+  box-sizing: border-box;
+  background-color: pink;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+/********FORM PARTS**********/
+
+form {
+  border: 1px solid #000;
+  width: 200px;
+  background-color: lightgray;
+  padding: 10px;
+  margin-bottom: 20px;
+  box-shadow: 8px 8px;
+  display: flex;
+  flex-direction: column;
+}
+
+.header-text {
+  font-size: 14px;
+  font-weight: bold;
+}
+
+.input {
+  height: 70px;
+  margin-bottom: 13px;
+  resize: vertical;
+}
+
+.form-button {
+  background-color: pink;
+  border: none;
+  border-radius: 25px;
+  height: 40px;
+  width: 206px;
+}
+
+/*******MESSAGE PART***********/
+
+.message-container {
+  border: 1px solid #000;
+  background-color: #fff;
+  width: 200px;
+  padding: 10px;
+  box-shadow: 8px 8px;
+  margin-bottom: 20px;
+  overflow-wrap: break-word;
+}
+
+.input-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+button {
+  border: none;
+  border-radius: 50%;
+  width: 40px;
+  height: 40px;
+  background-color: var(---buttons);
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+button:hover {
+  background-color: #e75480;
+}
+
+.zeroClicks {
+  background-color: lightgray;
+}
+
+.likes-container {
+  display: flex;
+  flex-direction: row;
```

Comment:
`flex-direction: row` is default for `display : flex`, so you don't need to specify it.